packages/apps/stock/src/screens/internalMoves/InternalMoveLineCreationScreen.js

```diff
@@ -68,14 +68,19 @@ const InternalMoveLineCreationScreen = ({navigation}) => {
 
   const handleToProductTrackingNumberChange = useCallback(
     _value => {
+      if (destinationStockLocation != null) {
+        setStockLocationLine(_value);
+        return;
+      }
+
```

Comment:
I don't understand the functional meaning of this. I don't think this is a good pratice too as you are skipping checks

Reply:
This is to skip the next step when we change the product, ensuring that the selected destination stock location is not lost.